readme.md

```diff
@@ -46,8 +46,70 @@ You can call our support team at (480) 505-8857
 
 Development of this plugin is done on [GitHub](https://github.com/godaddy/wp-reseller-store). If you believe you have found a bug, or have a killer feature idea, please open a [open a new issue](https://github.com/godaddy/wp-reseller-store/issues) there. Pull requests on existing issues are also welcome!
 
+## Frequently Asked Questions ##
+
+### What reseller programs will this plugin work with? ###
+This plugin is designed for the [GoDaddy Reseller Program](https://www.godaddy.com/reseller-program).  If you have not purchased a reseller program, you can activate the plugin with our "in house" reseller.  At any time, you can updgrade the reseller to your own. 
+
+### What settings should I update in the Reseller Control Center? ###
+We recommend you set the _Home Url_ to your WordPress storefront.  We also suggest you set up a _Custom Domain_ and point the DNS to our server.
+The _Custom Domain_ must be on a subdomain that matches the same root domain of your WordPress storefront.  For example, your WordPress storefront is hosted on _www_ and your custom domain is hosted on _shop_.  Follow these instructions to set your custom domain.  https://www.godaddy.com/help/use-a-custom-domain-for-my-storefront-5040
+
+You can update both the home url and custom domain setting at https://reseller.godaddy.com/settings/branding
+
+### How do I set the currency and language for the product catalog? ###
+The currency and language are set in the [Reseller Control Center](https://http://reseller.godaddy.com)
+
+### Do you have a pre-built WordPress site? ###
+We encourage each reseller to create their own unique site and content for their storefronts.  This provides a better experience for your customers and more opportunities for search engine optimization.
+While we can't test every theme with our plugin, we try our best to make the plugin compatible.  GoDaddy provides several free themes including [Primer](https://wordpress.org/themes/primer/) and [Activation](https://wordpress.org/themes/activation/) that we test with our plugin.  You can use content from the turnkey website to help you build your own WordPress site.
+
+### I am getting an error while trying to activate the plugin. What should I do? ###
+If you are having an issue, we recommend you deactivate your other plugins and then try to activate the Reseller Store plugin.  You can also try changing to another theme and the try re activating the plugin.  Before making any of these changes, we recommend you make a backup of your site and database.  
+
+While we strive to make the plugin compatible many themes and plugins, our plugin creates a custom post type and some themes and custom builder plugins can interfere with the registration of the custom post type.
```

Comment:
While we strive to make the plugin compatible **with** many themes and plugins, our plugin creates a custom post type, **which can cause** ~~and~~ some themes and custom builder plugins **to** ~~can~~ interfere with the registration of the custom post type.